src/main/kotlin/model/Dealer.kt

```diff
@@ -0,0 +1,25 @@
+package model
+
+class Dealer() : BasePlayer("Deal") {
+    private val deck = Deck()
```

Comment:
> though dealer is my default named "Dealer" from the constructor at the time of creating instance.

In a Blackjack game, the dealer's name doesn't seem to be an important part of the business logic.
It's more of a UI-level distinction between the player and the dealer.

I think u can remove `name` property from `BasePlayer` 🤔